src/6model/reprs.c

```diff
@@ -196,7 +196,7 @@ void MVM_repr_initialize_registry(MVMThreadContext *tc) {
 
     /* Add all core representations. */
     register_core_repr(String);
-    register_core_repr(Array);
+    register_repr(tc, VMArray_initialize(tc), NULL);
     register_core_repr(Hash);
```

Comment:
Check line 188:

#define register_core_repr(name) \
    register_repr(tc, MVM##name##_initialize(tc), NULL)

I expanded the macro manually because it forces MVM to be in there